config/config_test.go

```diff
@@ -16,10 +16,13 @@ package config
 
 import (
 	"errors"
-	"reflect"
+	"net/url"
 	"testing"
 
+	"github.com/stretchr/testify/assert"
```

Comment:
Any reason for this change? Should be included in commit message.

Reply:
Updated commit message to mention this and rationale for using it.